### PR TITLE
Created doc: TTL

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# TTL
+
+TTL indicates time to live. Use the [TTL options](#ttl_options) to specify a timeout for a vertex property or an edge property. Once the timeout is reached, the property expires.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md:3]: for a property? Because TTL is not for a single vertex, it's for a tag or edge type.

Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md:3]: Once the time is out?

File: docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Execution succeeded (time spent 1902/2642 us)
+```
+
+### Set a timeout when creating a tag or an edge type
+
+Use TTL options in the `CREATE` statement to set a timeout when creating a tag or an edge type. For more information, see [CREATE TAG](../10.tag-statements/1.create-tag.md) or [CREATE EDGE](../11.edge-type-statements/1.create-edge.md).
+
+```ngql
+// Create a tag and set the TTL options.
+CREATE TAG t2(a int, b int, c string) ttl_duration= 100, ttl_col = "a";
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md:72]: nebula>

**Review by Amber1990Zhang — APPROVED** [review]
minors

**Review by jude-zhu — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**